src/components/Modal/SbModalType.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <div>
+    <SbModal :is-open-modal="open" @hide="hide">
+      <SbModalHeader :icon="returnIconName" :title="title" :align="align" />
+      <SbModalContent>
+        {{ message }}
+      </SbModalContent>
+
+      <div>
+        <SbButton
+          type="ghost"
+          :label="cancelButtonLabel"
+          @click="handleCancelAction"
+        />
+        <SbButton
+          :type="returnButtonType"
+          :label="actionButtonLabel"
+          @click="handleDispatchAction"
+        />
+      </div>
+    </SbModal>
+  </div>
+</template>
+
+<script>
+import { SbModal, SbModalHeader, SbModalContent } from './index'
+import SbButton from '../Button'
+
+export default {
+  name: 'SbModalType',
+
+  components: {
+    SbModal,
+    SbModalHeader,
+    SbModalContent,
+    SbButton,
+  },
+
+  props: {
+    actionButtonLabel: {
+      type: String,
+      default: null,
+    },
+    align: {
+      type: String,
+      default: null,
+    },
+    cancelButtonLabel: {
+      type: String,
+      default: null,
+    },
+    message: {
+      type: String,
+      default: null,
+    },
+    title: {
+      type: String,
+      default: null,
+    },
+    type: {
+      type: String,
+      default: null,
+    },
+  },
+
+  data: () => ({
+    open: false,
+  }),
+
+  computed: {
+    returnIconName() {
+      return this.type === 'confirmation'
+        ? 'success-pictogram'
+        : 'delete-pictogram'
+    },
+
+    returnButtonType() {
+      return this.type === 'confirmation' ? 'primary' : 'danger'
+    },
+  },
+
+  methods: {
+    show() {
+      this.open = true
+    },
+
+    hide() {
+      this.open = false
+      this.$emit('on-hide')
```

Comment:
I believe that is a better name for the event just `hide`. And, this will be fix an expected event reference name in `create-modal-plugin.js` that instantiates a modal with a `hide` event ([you can check this here](https://github.com/storyblok/storyblok-design-system/blob/feature%2Fmodal-component/src/components/Modal/plugin/create-modal-plugin.js#L27))